public/app/frontend/src/components/navigation-main/navigation-main.scss

```diff
@@ -13,6 +13,7 @@
         width: 100%;
         @media screen and (min-width: $justice-lg) {
             flex-direction: row;
+            margin-left: -0.625rem;
```

Comment:
Could this be replaced with a new variable, or a a fraction of an existing variable like `$gutter / 2`?

Reply:
Done :)